cg/meta/archive/archive.py

```diff
@@ -34,12 +44,29 @@ class SpringArchiveAPI:
     """Class handling the archiving of sample SPRING files to an off-premise location for long
     term storage."""
 
-    def __init__(self, housekeeper_api: HousekeeperAPI, status_db: Store):
+    def __init__(
+        self,
+        ddn_dataflow_client: DDNDataFlowClient,
+        housekeeper_api: HousekeeperAPI,
+        status_db: Store,
+    ):
         self.housekeeper_api: HousekeeperAPI = housekeeper_api
+        self.ddn_client: DDNDataFlowClient = ddn_dataflow_client
         self.status_db: Store = status_db
+        self.handler_map: Dict[str, ArchiveModels] = {
+            ArchiveLocationsInUse.KAROLINSKA_BUCKET: ArchiveModels(
+                file_model=MiriaFile.from_file_and_sample, handler=self.ddn_client
+            )
+        }
+
+    def call_corresponding_archiving_function(
+        self, files: List[FileTransferData], archive_location: ArchiveLocationsInUse
```

Comment:
Out of scope for this PR, but `ArchiveLocationsInUse` makes it sound like there is an `ArchiveLocationsNotInUse`. Would it make sense to just call it `ArchiveLocations`?

Reply:
Yes, I think it would make sense, and if someone is trying to use one which is not in use we raise an error or a warning or something.

Reply:
Fixed